src/rtutils.c

```diff
@@ -339,13 +339,14 @@ static void jl_reserve_excstack(jl_excstack_t **stack JL_REQUIRE_ROOTED_SLOT,
     if (s)
         jl_copy_excstack(new_s, s);
     *stack = new_s;
+    jl_gc_wb(task, new_s);
```

Comment:
Could this WB be simply `jl_gc_wb(stack, new_s)`?

Reply:
`stack` is the address of the field in `task` (i.e. `&task.excstack`), while `task` is the object that is being written to. I think the write barrier should use `task`. 

However, as `stack` is always computed from `task`, we can actually remove the parameter `stack` and compute it from `task` in the function. I did not do this change, as I am not sure about the annotations on `jl_excstack_t **stack JL_REQUIRE_ROOTED_SLOT JL_ROOTING_ARGUMENT`.